src/3_PB/AnalisiDeiRequisiti/content.typ

```diff
@@ -90,55 +89,37 @@ Relativamente all'utilizzo della #glossary("dashboard"), viene definito un unico
   caption: [UC0 Visualizzazione menù #glossary("dashboard")]
 )
 
-/*TODO: ricontrollare la fine di ogni use case che comprende l'uso di un #glossary("pannello")*/
-=== UC1: Visualizzazione dashboard generale sensori
+=== UC1: Visualizzazione #glossary[dashboard] dati grezzi
 - *Attore Principale*: #glossary("amministratore pubblico")\;
 - *Precondizioni*: il #glossary("sistema") è operativo e accessibile;
-- *Postcondizioni*: l'#glossary("amministratore pubblico") visualizza #glossary("pannelli") relativi allo stato dei sensori, all'interno di un'unica #glossary[dashboard]\;
+- *Postcondizioni*: l'#glossary("amministratore pubblico") visualizza un #glossary("pannello") contenente i dati grezzi inviati dai sensori, all'interno di un'unica #glossary[dashboard]\;
 - *Scenario Principale*:
   + L'#glossary("amministratore pubblico") accede alla piattaforma di visualizzazione; 
-  + L'#glossary("amministratore pubblico") seleziona la visualizzazione della #glossary("dashboard") generale relativa ai sensori.
+  + L'#glossary("amministratore pubblico") seleziona la visualizzazione della #glossary("dashboard") relativa ai dati grezzi.
 
 #figure(
-  image("assets/UML/UC1_Visualizzazione-dashboard-generale-sensori.png",width:70%),
-  caption: [UC1 Visualizzazione #glossary("dashboard") generale sensori]
+  image("assets/UML/UC1_Visualizzazione-dashboard-dati-grezzi.png",width:70%),
+  caption: [UC1 Visualizzazione #glossary("dashboard") dati grezzi]
 )
 
-
-#pagebreak()
-=== UC1.1: Visualizzazione posizione sensori su mappa
+=== UC1.1: Visualizzazione #glossary("pannello") dati grezzi
 - *Attore Principale*: #glossary("amministratore pubblico")\;
-- *Precondizioni*: il #glossary("sistema") è operativo e accessibile;
-- *Postcondizioni*: l'#glossary("amministratore pubblico") visualizza un #glossary("pannello") contenente una mappa che mostra le posizioni dei sensori, visualizzati tramite icone, su di essa. Le icone dei sensori sono colorate in base al tipo di #glossary("sensore")  e riportano una label che ne esplicita il tipo, il nome e le coordinate geografiche; 
+- *Precondizioni*: l'#glossary("amministratore pubblico") ha selezionato la visualizzazione relativa al dominio dei dati grezzi;
+- *Postcondizioni*: l'#glossary("amministratore pubblico") visualizza il #glossary("pannello") relativo ai dati grezzi, che riporta il nome del #glossary[sensore], la tipologia del #glossary[sensore], il timestamp della rilevazione e il valore della misurazione, in forma tabellare, per ogni tipo di #glossary[sensore]; la tabella contenente i dati menzionati è indicizzata tramite il nome dei sensori; /*Nel caso in cui il #glossary[sensore] invii altri dati contestuali alla misurazione principale, come nel caso dei sensori che monitorano le colonne di ricarica [UC17], la congestione stradale [UC20] o il vento [UC12], questi saranno a loro volta visualizzabili all'interno della tabella.*/
```

Comment:
a questo proposito dato che per ogni sensore i dati mostrati sono i più di uno, (ti consiglio di dare un occhiata al prodotto), forse sarebbe da essere più specifici in qualche modo?

Reply:
lo stesso discorso da applicarsi al requisito corrispondente. (comunque andrei almeno a che quella è definita come misurazione può essere compostita)